app/client/rest/base.ts

```diff
@@ -171,6 +171,14 @@ export default class ClientBase extends ClientTracking {
         return `${this.urlVersion}/redirect_location`;
     }
 
+    getScheduledPostActionsRoute() {
```

Comment:
This is the same as `getScheduledPostRoute`, so can we remove this?

Reply:
Sure